src/odemis/driver/test/actuator_test.py

```diff
@@ -1117,30 +1122,50 @@ def test_offset_moveAbs(self):
         f = self.dev_cycle.moveAbs({axis_name: 0.0})
         f.result()
 
-        new_pos = 1.5707963267948966  # pi/2
-        for i in range(1, 6):
-            # overrun 2pi after 4 moves in clockwise
-            f = self.dev_cycle.moveRel({axis_name: new_pos})
+        # Check it runs referencing also after many small moves, by checking the time
+        durations = []
+        for i in range(self.ref_freq):
+            start_t = time.time()
+            f = self.dev_cycle.moveRelSync({axis_name: 0.1})
+            durations.append(time.time() - start_t)
+
+        # It should have 1 move which takes quite a lot longer (+1s)
+        dur_outlier = max(durations)
```

Comment:
I'd rather not call it "dur_ref_move", because I don't know if it's actually due to referencing. That's what we are testing, and hoping. I'll call it just "dur_longer".